<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinImageFilter</class>
 <widget class="QWidget" name="WinImageFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinImageFilterOptions</string>
  </property>
  <property name="description" stdset="0">
   <string>Select the file wich you want to process.
In the grid option move the lower and upper color border until you see only the grid of your symbol image.
Make sure the lines forms all cross points of the grid.
Optional you can apply a symbol filter to extract the lines in the same way.</string>
  </property>
  <property name="navButtons" stdset="0">
   <UInt>11</UInt>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="listViewFiles">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabWidgetPageGrid">
          <property name="id" stdset="0">
           <number>0</number>
          </property>
          <attribute name="title">
           <string>Grid</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxGridAutoRotate">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Auto Rotate</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBoxGridInvertMask">
             <property name="text">
              <string>Inverted Mask</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonGridImageOriginal">
             <property name="text">
              <string>Original image</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="id" stdset="0">
              <number>0</number>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupGridPreviewSelection</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButtonGridImageMask">
             <property name="text">
              <string>Grid image mask</string>
             </property>
             <property name="id" stdset="0">
              <number>1</number>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupGridPreviewSelection</string>
             </attribute>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="radioButtonGridImage">
             <property name="text">
              <string>Grid image</string>
             </property>
             <property name="id" stdset="0">
              <number>2</number>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupGridPreviewSelection</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabWidgetPageSymbol">
          <property name="id" stdset="0">
           <number>1</number>
          </property>
          <attribute name="title">
           <string>Symbol</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxSymbolUseMask">
             <property name="text">
              <string>Use Mask</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBoxSymbolInvertMask">
             <property name="text">
              <string>Inverted Mask</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonSymbolImageOriginal">
             <property name="text">
              <string>Original image</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="id" stdset="0">
              <number>0</number>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupSymbolPreviewSelection</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButtonSymbolMask">
             <property name="text">
              <string>Symbol image mask</string>
             </property>
             <property name="id" stdset="0">
              <number>1</number>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupSymbolPreviewSelection</string>
             </attribute>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="radioButtonSymbolImage">
             <property name="text">
              <string>Symbol image</string>
             </property>
             <property name="id" stdset="0">
              <number>2</number>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroupSymbolPreviewSelection</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetBorders" native="true">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxLower">
         <property name="minimumSize">
          <size>
           <width>331</width>
           <height>231</height>
          </size>
         </property>
         <property name="title">
          <string>Lower Border</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="WidgetColorSelector" name="widgetLowerColorSelector" native="true">
            <property name="color" stdset="0">
             <color>
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxUpper">
         <property name="minimumSize">
          <size>
           <width>331</width>
           <height>231</height>
          </size>
         </property>
         <property name="title">
          <string>Upper Border</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="WidgetColorSelector" name="widgetUpperColorSelector" native="true">
            <property name="color" stdset="0">
             <color>
              <red>240</red>
              <green>155</green>
              <blue>157</blue>
             </color>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxPreview">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QAspectRatioPixmapLabel" name="labelPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>No image selected</string>
         </property>
         <property name="loadingText" stdset="0">
          <string>Process image ...</string>
         </property>
         <property name="defaultText" stdset="0">
          <string>No image selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAspectRatioPixmapLabel</class>
   <extends>QLabel</extends>
   <header>QAspectRatioPixmapLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetColorSelector</class>
   <extends>QWidget</extends>
   <header>WidgetColorSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupGridPreviewSelection"/>
  <buttongroup name="buttonGroupSymbolPreviewSelection"/>
 </buttongroups>
</ui>
